src/main/java/io/quarkus/bot/release/ReleaseInformation.java

```diff
@@ -56,14 +62,24 @@ public String getFullBranch() {
         return branch;
     }
 
+    /**
+     * @return the branch from which we branched - it is main except when preparing the CR1 of an LTS release, in which case it
+     *         is the previous version branch as we want to branch from the previous version branch
+     */
     public String getOriginBranch() {
         return originBranch;
     }
 
+    /**
+     * @return the qualifier (Alpha1, CR1, CR2...)
```

Comment:
For a Final version, what will this be? Empty string?

Reply:
We used to have the `Final` qualifier but nowadays, it's null, will add something about it.

Reply:
Keep in mind you have `isFinal()`.